docs/development/guidelines.md

```diff
@@ -258,6 +274,49 @@ Passing values that are commonly used together around separately is tedious, ver
     training_feature_vectors, validation_feature_vectors, training_target_values, validation_target_values = split(feature_vectors, target_values)
     ```
 
+## Code style
```

Comment:
Better move this to the end of the document:

1. Design
2. Docstrings
3. Tests
4. Code style

Code style is the least important concern for our users, so it's also not as important for developers to keep in mind as the other items.